src/parser/jobs/sch/index.js

```diff
@@ -19,8 +19,8 @@ export const SCHOLAR = new Meta({
 	</>,
 
 	supportedPatches: {
-		from: '✖',
-		to: '✖',
+		from: '7.0',
```

Comment:
These data changes aren't sufficient to be ready for support - we should at least add some handling for making sure people are using Impact Imminent, handle the effects of Seraphism on Emergency Tactics